BRANDING.md

```diff
@@ -0,0 +1,55 @@
+# Istio Brand Guidelines
```

Comment:
It seems like there may be a lot of usages in the wild already that may not match the guidelines here. Do we have a plan on what will be done with these?

Reply:
We're aware of some uses already and are in process of addressing them. For others that are brought to Steering's attention, we'll review and determine if usage is okay or not. The main goal for trademark is to protect consumers so they don't have to question what's really part of Istio or not.